include/File.hpp

```diff
@@ -0,0 +1,16 @@
+#ifndef _coffeemaker_file_hpp
+#define _coffeemaker_file_hpp
+
+#include <string>
+
+namespace CoffeeMaker {
+
+  struct File {
+    std::string name;
+    bool loaded;
+    char* data;
```

Comment:
This will leak memory in the event of a successful file read. Need a way to clean up the memory.